corehq/apps/userreports/sql/adapter.py

```diff
@@ -34,10 +37,34 @@ def __init__(self, config):
     def get_table(self):
         return get_indicator_table(self.config)
 
+    @memoized
+    def get_sqlalchemy_mapping(self):
+        table = self.get_table()
+        Base = declarative_base(metadata=metadata)
+        properties = dict(table.columns)
+        properties['__tablename__'] = table.name
+        properties['__table_args__'] = ({'extend_existing': True},)
```

Comment:
@millerdev fyi this is what I was asking about on Friday. I ended up having to step through the sqlalchemy code to find it